README.md

```diff
@@ -3,59 +3,66 @@
-* [pycma](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#pycma)
-* [SHAP](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#shap)
-* [sklearn](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#sklearn)  ([example](https://github.com/optuna/optuna-examples/tree/main/sklearn/sklearn_optuna_search_cv_simple.py))
-* [skorch](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#skorch)  ([example](https://github.com/optuna/optuna-examples/tree/main/pytorch/skorch_simple.py))
-* [TensorBoard](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#tensorboard)  ([example](https://github.com/optuna/optuna-examples/tree/main/tensorboard/tensorboard_simple.py))
-* [tf.keras](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#tensorflow)  ([example](https://github.com/optuna/optuna-examples/tree/main/tfkeras/tfkeras_integration.py))
-* [Weights & Biases](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#wandb)  ([example](https://github.com/optuna/optuna-examples/blob/main/wandb/wandb_integration.py))
-* [XGBoost](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#xgboost)  ([example](https://github.com/optuna/optuna-examples/tree/main/xgboost/xgboost_integration.py))
+This package is an integration module of [Optuna](https://github.com/optuna/optuna), an automatic Hyperparameter optimization software framework.
+The modules in this package provide users with extended functionalities for Optuna in combination with third-party libraries such as PyTorch, sklearn, and TensorFlow.
+
+> [!NOTE]
+> You can find more information in [**our official documentations**](https://optuna-integration.readthedocs.io/en/stable/) and [**API reference**](https://optuna-integration.readthedocs.io/en/stable/reference/index.html).
 
 ## Installation
 
-Optuna-Integration is available at [the Python Package Index](https://pypi.org/project/optuna-integration/) and
-on [Anaconda Cloud](https://anaconda.org/conda-forge/optuna-integration).
+Optuna-Integration is available via [pip](https://pypi.org/project/optuna-integration/) and
+on [conda](https://anaconda.org/conda-forge/optuna-integration).
 
 ```bash
 # PyPI
 $ pip install optuna-integration
-```
 
-```bash
 # Anaconda Cloud
 $ conda install -c conda-forge optuna-integration
 ```
 
-Optuna-Integration supports from Python 3.7 to Python 3.10.
-
-Also, we also provide Optuna docker images on [DockerHub](https://hub.docker.com/r/optuna/optuna).
+> [!IMPORTANT]
+> As dependencies of all the modules are large and complicated, the commands above install only the common dependencies.
+> Dependencies for each module are described in `requirements.txt` in the corresponding directory and we kindly ask users to separately install them.
+
+> [!NOTE]
+> Optuna-Integration supports from Python 3.7 to Python 3.11.
```

Comment:
It might be good to update the PyPI classifier in the following pull request.
https://github.com/optuna/optuna-integration/blob/110ce7d57d3ea13595e393df73a675a46ee7e009/pyproject.toml#L19

Reply:
I've just opened https://github.com/optuna/optuna-integration/pull/129